buildimage.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+sudo debootstrap focal focal > /dev/null
```

Comment:
what is focal?

Reply:
Focal is the update of Ubuntu I plan on using for the Docker images. It is the most recent LTS version of Ubuntu, so it will receive security and stability support for a few years. If you have a different Ubuntu update that would fit this project better, please let me know.